tests/test_modules.c

```diff
@@ -140,12 +140,12 @@ test_install_module(void **state)
             "<module xmlns=\"http://www.sysrepo.org/yang/sysrepo\">"
             "<name>main-mod</name>"
             "<enabled-feature>feat</enabled-feature>"
-            "<plugin><datastore>ds:startup</datastore><name>JSON DS file</name></plugin>"
-            "<plugin><datastore>ds:running</datastore><name>JSON DS file</name></plugin>"
             "<plugin><datastore>ds:candidate</datastore><name>JSON DS file</name></plugin>"
-            "<plugin><datastore>ds:operational</datastore><name>JSON DS file</name></plugin>"
             "<plugin><datastore>fd:factory-default</datastore><name>JSON DS file</name></plugin>"
             "<plugin><datastore>notification</datastore><name>JSON notif</name></plugin>"
+            "<plugin><datastore>ds:operational</datastore><name>JSON DS file</name></plugin>"
+            "<plugin><datastore>ds:running</datastore><name>JSON DS file</name></plugin>"
+            "<plugin><datastore>ds:startup</datastore><name>JSON DS file</name></plugin>"
```

Comment:
Looking at this I am wondering whether identities should not be ordered based on their order in the module, it certainly expresses something in this case (from the simplest and most obvious to more obscure ones). Ideally, the inheritance hierarchy should also be expressed in "DFS" order. But I am not sure it is worth it, would probably not be that simple and may potentially take time to figure out.